SocialFilter/sections/specific.txt

```diff
@@ -1133,7 +1133,7 @@ newyorker.com##.cne-video-buttons
 denofgeek.com##.social-links__items
 denofgeek.com##.social-links__items + span.sep
 goldapple.ru##.pdp__share
-koronawirus.lol,kapwing.com,stopgame.ru,dailywire.com,donald.pl,speed.cloudflare.com,hackernoon.com##.react-share__ShareButton
+futurism.com,koronawirus.lol,kapwing.com,stopgame.ru,dailywire.com,donald.pl,speed.cloudflare.com,hackernoon.com##.react-share__ShareButton
```

Comment:
We need to hide "SHARE THIS ARTICLE" inscription too.
In both cases it is better to use extended CSS.

Reply:
any recommendations?

Reply:
`futurism.com#?#main > div > div.text-center:has(> div.flex > button.react-share__ShareButton)`

and for the newsletter block:
`futurism.com#?##__next > div.antialiased > div > div.text-center:has(> div.mx-auto > div:contains(newsletter))
`